fastembed/embedding.py

```diff
@@ -126,26 +149,37 @@ def onnx_embed(self, documents: List[str]) -> np.ndarray:
             )
 
         model_output = self.model.run(None, onnx_input)
-        last_hidden_state = model_output[0][:, 0]
+        last_hidden_state = self._pooling_fn(model_output[0], attention_mask)
         embeddings = normalize(last_hidden_state).astype(np.float32)
```

Comment:
With Jina, do we need to normalize again? I believe we're already implementing mean pooling in a separate function — so we should handle pooling separately

Reply:
I can't find pooling being defined anywhere else